cpp/src/search/search.cu

```diff
@@ -210,6 +212,62 @@ bool contains(column_view const& col,
                                              col, value,
                                              stream, mr);
 }
+
+struct multi_contains_dispatch {
+  template <typename Element>
+  void operator()(column_view const& col,
+                  column_view const& in_col,
+                  mutable_column_view &result_view,
+                  cudaStream_t stream) {
+
+    auto hash_set = fixed_hash_set<Element>::create(in_col, stream);
+    auto device_hash_set = hash_set.to_device();
+
+    auto d_col_ptr = column_device_view::create(col, stream);
+    auto d_col = *d_col_ptr;
+
+    thrust::transform(rmm::exec_policy(stream)->on(stream),
+                      thrust::make_counting_iterator<size_type>(0),
+                      thrust::make_counting_iterator<size_type>(col.size()),
+                      result_view.begin<bool8>(),
+                      [device_hash_set, d_col] __device__ (size_t index) {
+                        return d_col.is_null(index) || device_hash_set.contains(d_col.element<Element>(index));
```

Comment:
Add a template bool arg to this function object that allows omitting the `is_null` check if the input doesn't actually contain null values.

Reply:
Done